src/games/strategy/triplea/ui/NotesPanel.java

```diff
@@ -1,20 +1,15 @@
 package games.strategy.triplea.ui;
 
-import java.awt.Component;
 import java.awt.Rectangle;
 
 import javax.swing.BoxLayout;
-import javax.swing.JButton;
 import javax.swing.JEditorPane;
-import javax.swing.JLabel;
 import javax.swing.JPanel;
 import javax.swing.JScrollPane;
-import javax.swing.SwingUtilities;
 
 public class NotesPanel extends JPanel {
   private static final long serialVersionUID = 2746643868463714526L;
   protected final JEditorPane m_gameNotesPane;
-  final JButton m_refresh = new JButton("Refresh Notes");
```

Comment:
That was a pretty large across the board refactor, in theory should not have changed this behavior. Could you be more specific what changed to break the refresh button?

I'm not a fan of the placement of the refresh button, nor that it is always visible. IMO it should only be available in a "map-making" mode. If we have any map making tools that render the refresh button obsolete, removing it from the game UI would be a good move in my judgement.